code/game/objects/objs.dm

```diff
@@ -20,7 +20,7 @@
 	///Damage under this value will be completely ignored
 	var/damage_deflection = 0
 
-	var/resistance_flags = NONE // INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ON_FIRE | UNACIDABLE | ACID_PROOF
+	var/resistance_flags = NONE // INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ON_FIRE | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
```

Comment:
if ya could, please revert the added comment. Since its in the core files it could possibly one day cause a conflict for no reason.
Merging when the comment's removed